wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/WorkspaceDao.java

```diff
@@ -142,4 +142,21 @@
      *         when any other error occurs during workspaces fetching
      */
     List<WorkspaceImpl> getWorkspaces(String userId) throws ServerException;
+
+
+    /**
+     * Gets workspaces by temporary attribute.
```

Comment:
please add the doc which describes exceptions in situation when
`maxItems` and `skipCount` are invalid.